### ISSUE TITLE
Is a bug in 'Many-to-Many' relations?

### ISSUE THREAD

**Issue by srisy** [balderdashy/sails#2166]
I follow the 'Many-to-Many' case at 
[http://sailsjs.org/#/documentation/concepts/ORM/Associations/ManytoMany.html](http://sailsjs.org/#/documentation/concepts/ORM/Associations/ManytoMany.html),
when i run the app to test,via ```sails console```, the output is different from the page, 
I encounter the same problem either than `sails-mysql` or `sails-mongo`,
see it:
```

E:\My Projects\SailsDemo>sails console

info: Starting app in interactive mode...

Associating  Rainbow Dash with Gabe
Associating  Rainbow Dash with Cody
Associating  Rainbow Dash with Mike
Done with pony  Rainbow Dash
Associating  Pinkie Pie with Gabe
Associating  Pinkie Pie with Cody
Associating  Pinkie Pie with Mike
Done with pony  Pinkie Pie
Associating  Applejack with Gabe
Associating  Applejack with Cody
Associating  Applejack with Mike
Done with pony  Applejack
Everyone belongs to everyone!! Exiting.
info: Welcome to the Sails console.
info: ( to exit, type <CTRL>+<C> )

sails> null { pets:
   [ { name: 'Applejack',
       color: 'orange',
       id: 1,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Applejack',
       color: 'orange',
       id: 1,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Applejack',
       color: 'orange',
       id: 1,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Pinkie Pie',
       color: 'pink',
       id: 2,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Pinkie Pie',
       color: 'pink',
       id: 2,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Pinkie Pie',
       color: 'pink',
       id: 2,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Rainbow Dash',
       color: 'blue',
       id: 3,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Rainbow Dash',
       color: 'blue',
       id: 3,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Rainbow Dash',
       color: 'blue',
       id: 3,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) } ],
  name: 'Mike',
  age: 16,
  id: 1,
  createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
  updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) }
null { pets:
   [ { name: 'Applejack',
       color: 'orange',
       id: 1,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Applejack',
       color: 'orange',
       id: 1,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Applejack',
       color: 'orange',
       id: 1,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Pinkie Pie',
       color: 'pink',
       id: 2,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Pinkie Pie',
       color: 'pink',
       id: 2,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Pinkie Pie',
       color: 'pink',
       id: 2,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Rainbow Dash',
       color: 'blue',
       id: 3,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Rainbow Dash',
       color: 'blue',
       id: 3,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Rainbow Dash',
       color: 'blue',
       id: 3,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) } ],
  name: 'Cody',
  age: 25,
  id: 2,
  createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
  updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) }
null { pets:
   [ { name: 'Applejack',
       color: 'orange',
       id: 1,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Applejack',
       color: 'orange',
       id: 1,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Applejack',
       color: 'orange',
       id: 1,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Pinkie Pie',
       color: 'pink',
       id: 2,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Pinkie Pie',
       color: 'pink',
       id: 2,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Pinkie Pie',
       color: 'pink',
       id: 2,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Rainbow Dash',
       color: 'blue',
       id: 3,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Rainbow Dash',
       color: 'blue',
       id: 3,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Rainbow Dash',
       color: 'blue',
       id: 3,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) } ],
  name: 'Mike',
  age: 16,
  id: 1,
  createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
  updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) }
null { pets:
   [ { name: 'Applejack',
       color: 'orange',
       id: 1,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Applejack',
       color: 'orange',
       id: 1,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Applejack',
       color: 'orange',
       id: 1,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Pinkie Pie',
       color: 'pink',
       id: 2,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Pinkie Pie',
       color: 'pink',
       id: 2,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Pinkie Pie',
       color: 'pink',
       id: 2,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Rainbow Dash',
       color: 'blue',
       id: 3,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Rainbow Dash',
       color: 'blue',
       id: 3,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Rainbow Dash',
       color: 'blue',
       id: 3,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) } ],
  name: 'Gabe',
  age: 107,
  id: 3,
  createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
  updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) }
null { pets:
   [ { name: 'Applejack',
       color: 'orange',
       id: 1,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Applejack',
       color: 'orange',
       id: 1,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Applejack',
       color: 'orange',
       id: 1,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Pinkie Pie',
       color: 'pink',
       id: 2,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Pinkie Pie',
       color: 'pink',
       id: 2,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Pinkie Pie',
       color: 'pink',
       id: 2,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Rainbow Dash',
       color: 'blue',
       id: 3,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Rainbow Dash',
       color: 'blue',
       id: 3,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Rainbow Dash',
       color: 'blue',
       id: 3,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) } ],
  name: 'Cody',
  age: 25,
  id: 2,
  createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
  updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) }
null { pets:
   [ { name: 'Applejack',
       color: 'orange',
       id: 1,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Applejack',
       color: 'orange',
       id: 1,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Applejack',
       color: 'orange',
       id: 1,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Pinkie Pie',
       color: 'pink',
       id: 2,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Pinkie Pie',
       color: 'pink',
       id: 2,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Pinkie Pie',
       color: 'pink',
       id: 2,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Rainbow Dash',
       color: 'blue',
       id: 3,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Rainbow Dash',
       color: 'blue',
       id: 3,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Rainbow Dash',
       color: 'blue',
       id: 3,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) } ],
  name: 'Mike',
  age: 16,
  id: 1,
  createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
  updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) }
null { pets:
   [ { name: 'Applejack',
       color: 'orange',
       id: 1,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Applejack',
       color: 'orange',
       id: 1,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Applejack',
       color: 'orange',
       id: 1,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Pinkie Pie',
       color: 'pink',
       id: 2,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Pinkie Pie',
       color: 'pink',
       id: 2,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Pinkie Pie',
       color: 'pink',
       id: 2,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Rainbow Dash',
       color: 'blue',
       id: 3,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Rainbow Dash',
       color: 'blue',
       id: 3,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Rainbow Dash',
       color: 'blue',
       id: 3,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) } ],
  name: 'Gabe',
  age: 107,
  id: 3,
  createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
  updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) }
null { pets:
   [ { name: 'Applejack',
       color: 'orange',
       id: 1,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Applejack',
       color: 'orange',
       id: 1,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Applejack',
       color: 'orange',
       id: 1,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Pinkie Pie',
       color: 'pink',
       id: 2,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Pinkie Pie',
       color: 'pink',
       id: 2,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Pinkie Pie',
       color: 'pink',
       id: 2,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Rainbow Dash',
       color: 'blue',
       id: 3,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Rainbow Dash',
       color: 'blue',
       id: 3,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Rainbow Dash',
       color: 'blue',
       id: 3,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) } ],
  name: 'Gabe',
  age: 107,
  id: 3,
  createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
  updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) }
null { pets:
   [ { name: 'Applejack',
       color: 'orange',
       id: 1,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Applejack',
       color: 'orange',
       id: 1,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Applejack',
       color: 'orange',
       id: 1,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Pinkie Pie',
       color: 'pink',
       id: 2,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Pinkie Pie',
       color: 'pink',
       id: 2,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Pinkie Pie',
       color: 'pink',
       id: 2,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Rainbow Dash',
       color: 'blue',
       id: 3,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Rainbow Dash',
       color: 'blue',
       id: 3,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) },
     { name: 'Rainbow Dash',
       color: 'blue',
       id: 3,
       createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
       updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) } ],
  name: 'Cody',
  age: 25,
  id: 2,
  createdAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间),
  updatedAt: Fri Aug 29 2014 17:20:44 GMT+0800 (中国标准时间) }
```

why a pet is add 3 times to the table `pet_owners__user_pets`???
see the table data:

**Comment by mahadazad** [comment]
Any updated on it? It seems broken to me:

https://github.com/balderdashy/sails-mongo/issues/316

**Comment by sailsbot** [comment]
Thanks for posting, @srisy.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
